CHANGELOG.md

```diff
@@ -1,10 +1,29 @@
 # Change Log
 
-## Version 1.36.1
-* Release date: April 22, 2022
+## Version 1.37.0
+* Release date: June 15, 2022
 * Release status: General Availability
 ## What's new in this version
-- April Hotfix addressing these issues https://github.com/microsoft/azuredatastudio/milestone/88?closed=1.
+* New Features:
+    *  Backup & Restore - Backup & Restore to URL is now available in preview for Azure SQL Managed Instances.
+    *  Table Designer - Added API to support computed column capabilities on Table Designer.
+    *  Table Designer - Can now specify where to add new columns and columns can now be re-arranged by mouse dragging.
+    *  Table Designer - Table Designer is now supported by SQL Projects to add or modify database schema without need to be connected to a server instance.
+    *  Query Plan Viewer - Smart plan comparison is now available. Can now compare execution plans and view detailed differences between plans in the Properties Table.
+    *  Query Plan Viewer - Added toggle button to switch between estimated and actual execution plans.
+    *  Query Plan Viewer - Query Plan now comes with improved precision to operator costs for larger plans.
+    *  SQL Bindings - Support for Azure SQL Bindings is now available in preview for C#, JavaScript, and Python.
```

Comment:
Remove this line - sql bindings aren't a part of ADS

Reply:
okay, I had copied this from Timi's blog post, should I also remove this line from our release page?

Reply:
That would be great! Thank you!